uk.co.piggz.amazfish.json

```
@@ -0,0 +1,154 @@
+        {
+          "type": "archive",
+          "url": "https://github.com/sailfishos/nemo-qml-plugin-dbus/archive/refs/tags/2.1.33.tar.gz",
+          "sha256": "08d1bdc5eac5eba2197f2c1e04a7e851437724331215059db1d68828dbf3b0a8"
+        },
+        {
+          "type": "patch",
+          "path": "nemodbus-qmake.patch"
+        }
+      ]
+    },
+    {
+      "name": "qtmpris",
+      "buildsystem": "qmake",
+      "builddir": true,
+      "cleanup": [
+        "/include"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/sailfishos/qtmpris/archive/refs/tags/1.0.6.tar.gz",
+          "sha256": "384eac76ec7d29d1de6fa93c45007ccd506d6cc0e4c69e19d4250e16cc0f34a2"
+        },
+        {
+          "type": "patch",
+          "path": "qtmpris-qmake.patch"
+        }
+      ]
+    },
+    {
+      "name": "libkdb3",
+      "buildsystem": "cmake",
+      "builddir": true,
+      "cleanup": [
+        "/include",
+        "/mkspecs"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://invent.kde.org/libraries/kdb/-/archive/tags/v3.2.0/kdb-v3.2.0.tar.gz",
+          "sha256": "3a5c660f0080c6971ac1046be84a0accd86a5296e9df4c585b1b91361b5eed20"
+        },
+        {
+          "type": "patch",
+          "path": "6.patch"
+        },
+        {
+          "type": "patch",
+          "path": "upstream_Fix-build-with-newer-Qt.patch"
+        },
+        {
+          "type": "patch",
+          "path": "libkdb3-features.patch"
+        }
+      ]
+    },
+    {
+      "name": "extra-cmake-modules",
+      "buildsystem": "cmake-ninja",
+      "cleanup": [
+         "*"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://invent.kde.org/frameworks/extra-cmake-modules/-/archive/v5.115.0/extra-cmake-modules-v5.155.0.tar.gz",
+          "sha256": "b63a9376786c29487f9b513df06b08ecda534852ae09d2e045f35e4e32dc1433"
+        }
+      ]
+    },
+    {
+      "name": "bluezqt",
+      "buildsystem": "cmake",
+      "builddir": true,
+      "cleanup": [
+        "/include",
+        "/mkspecs"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://invent.kde.org/frameworks/bluez-qt/-/archive/v5.115.0/bluez-qt-5.115.0.tar.gz",
+          "sha256": "e0c5012e2518d6ade9eefcff118b1f71b96a40a12396d07df798ce5afca7f3fc"
+        },
+        {
+          "type": "patch",
+          "path": "bluez-install-run.patch"
+        }
+      ]
+    },
+    {
+      "name": "harbour-amazfish",
+      "buildsystem": "qmake",
+      "config-opts": [
+        "FLAVOR=qtcontrols"
+      ],
+      "build-options": {
+        "env": [
+          "QMAKEPATH=/app"
+        ]
+      },
+      "post-install": [
+        "install -Dm755 run-amazfish.sh /app/bin/run-amazfish.sh",
+        "install -Dm644 ui/icons/128x128/harbour-amazfish-ui.png /app/share/icons/hicolor/128x128/apps/uk.co.piggz.amazfish.png",
```

```suggestion
        "install -Dm644 ui/icons/128x128/harbour-amazfish-ui.png -t /app/share/icons/hicolor/128x128/apps/",
```

you do the `rename-icon` so no need to rename here.

---

installing with original name